content/docs/wiki/cc3200/debug-port.md

```diff
@@ -64,7 +67,7 @@ Please connect the toniebox to your power supply and/or battery. Please double c
 | Toniebox | Toniebox | UART |
 | -------- | -------  | ---- |
 | GND      |          | GND  |
-| TX       |          | RX   |
+| TX       |          | RX   |      
```

Comment:
Please remove this additional whitespace

Reply:
I cleaned up the branch.